### DISCUSSION TITLE
Questions about placeholders in ThingModel

### DISCUSSION THREAD

**Discussion by fix-all** [eclipse-ditto/ditto#1914] [General]
hi,
due to different domain names in different environments, can I use placeholders to replace domain names in the 'tm:submodel' and 'tm:extends' href attributes?
Based on my testing, this is not allowed. The href I set looks like this: 
"links": [
    {
      "href": "{{WOT_MODELS_URL}}/base_model-0.0.1.tm.jsonld",
      "rel": "tm:extends",
      "type": "application/tm+json"
    }
]

**Comment by thjaeckle** [comment]
I think the placeholders can only be used to derive a TD from a TM - that's also how I understood the WoT standard:
https://www.w3.org/TR/wot-thing-description11/#thing-model-td-placeholder

**Reply by fix-all** [reply]
Yes, I used custom placeholders for this.

**Reply by thjaeckle** [reply]
Yes, they are not intended to be resolved in the TM, as far as I understand the standard.
Only when generating the TD from the TM.

But I guess that would make sense to resolve such globally configured  placeholders. Feel free to open an issue, we would accept a PR providing this.
